src/internal/connector/onedrive/collections.go

```diff
@@ -445,6 +471,32 @@ func (c *Collections) Get(
 	return collections, canUsePreviousBackup, nil
 }
 
+// addURLCacheToDriveCollections adds a URL cache to all collections belonging to
+// a drive.
+func (c *Collections) addURLCacheToDriveCollections(
```

Comment:
```suggestion
func (c *Collections) setURLCache(
```

Reply:
I prefer the original. It's more explicitly named. Also, `setURLCache` might also mean a (get)/set on URL cache.